src/chromium/setup.md

```diff
@@ -0,0 +1,22 @@
+# Setup
+
+Make sure you can build and run Chromium. Any platform and set of build flags is
+OK, so long as your code is relatively recent (commit position 1223636 onwards,
+corresponding to November 2023):
+
+```shell
+gn gen out/Debug
+autoninja -C out/Debug chrome
+out/Debug/chrome # or on Mac, out/Debug/Chromium.app/Contents/MacOS/Chromium
+```
+
+See [How to build Chromium](https://www.chromium.org/developers/how-tos/get-the-code/)
+if you aren't already at that point. Be warned - setting up to build Chromium
+takes time.
```

Comment:
Should there be a separate "prerequisites" section somewhere to make sure the attendees some prepared?